### PR TITLE
Bug 1125780 - [Stingray] Use event name to check if we need to invoke _c...

### PR DESCRIPTION
**Pull request by huchengtw-moz**
...losing handler

### CODE REVIEW

**Comment by try-server-hook** [comment]
[huchengtw-moz John Hu (huchengtw-moz)](https://github.com/huchengtw-moz) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=76ed0afb1914)

**Comment by try-server-hook** [comment]
[huchengtw-moz John Hu (huchengtw-moz)](https://github.com/huchengtw-moz) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=bbb5e4ff0b08)

**Comment by try-server-hook** [comment]
[huchengtw-moz John Hu (huchengtw-moz)](https://github.com/huchengtw-moz) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=bbad3f6ecfb7)

File: tv_apps/smart-system/js/child_window_factory.js [hunk 1]
```diff
@@ -40,12 +40,13 @@
...
     function cwf_handleEvent(evt) {
-      // Handle event from child window.
-      if (evt.detail && evt.detail.instanceID &&
-          evt.detail.instanceID !== this.app.instanceID) {
-        if (this['_handle_child_' + evt.type]) {
-          this['_handle_child_' + evt.type](evt);
-        }
+      // ChildWindowFactory handles window.open and activities. It listens the
+      // closing event on the element of PopupWindow and ActivityWindow. Once
+      // receives _closing event, we can say this event is fired by one of
```
Comment (luke-chang) [inline tv_apps/smart-system/js/child_window_factory.js]: "receiving"

**Comment by try-server-hook** [comment]
[huchengtw-moz John Hu (huchengtw-moz)](https://github.com/huchengtw-moz) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=92fafa9d4df2)

**Comment by try-server-hook** [comment]
[huchengtw-moz John Hu (huchengtw-moz)](https://github.com/huchengtw-moz) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=3089c9d1f6f4)

**Pull request merged**